tests/test_heaven.py

```
@@ -0,0 +1,50 @@
+from curses import wrapper
+from divine import *
```

Ruff (0.8.2)

`from divine import *` used; unable to detect undefined names

(F403)